docs/4___co-simulation.adoc

```diff
@@ -2,7 +2,7 @@
 
 FMI for Co-Simulation provides a standardized interface for execution of simulation models or tools in a co-simulation environment.
 In contrast to FMI for Model Exchange, a Co-Simulation FMU implements not only the model algorithm, but also the required solution method.
-The data exchange between FMUs is restricted to discrete communication points latexmath:[\mathbf{t}_i] (unless <<IntermediateUpdateMode>> is used).
```

Comment:
Maybe bold here as we talk about points not point or use some expression e.g  t_i  i E N, but maybe not so important

Reply:
I would leave it as there is an index here as used everywhere else to indicate the same.